Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsFragmentPresenter.java

```diff
@@ -897,6 +897,9 @@ private void addStatisticsSettings(ArrayList<SettingsItem> sl)
             R.string.show_vtimes_description));
     sl.add(new SwitchSetting(mContext, BooleanSetting.GFX_SHOW_GRAPHS, R.string.show_graphs,
             R.string.show_graphs_description));
+    sl.add(new IntSliderSetting(mContext, IntSetting.GFX_PERF_SAMP_WINDOW,
```

Comment:
It might look neater to put this below the SwitchSettings to avoid mixing setting types. What do you think?

Reply:
I like it